src/main/java/com/cinema/dao/MovieSessionDao.java

```diff
@@ -8,4 +8,6 @@
     List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
 
     MovieSession add(MovieSession session);
+
+    MovieSession findById(Long id);
```

Comment:
```suggestion
    MovieSession get(Long id);
```
uniform method names